be/src/exec/olap_scan_node.cpp

```diff
@@ -702,8 +694,13 @@ Status OlapScanNode::start_scan_thread(RuntimeState* state) {
                 state, this, _olap_scan_node.is_preaggregation, _need_agg_finalize, *scan_range, scanner_ranges);
             _scanner_pool->add(scanner);
             _olap_scanners.push_back(scanner);
+
+            if (disk_set.find(scanner->scan_disk()) == disk_set.end()) {
+                disk_set.insert(scanner->scan_disk());
```

Comment:
`disk_set` is a set, so no need to `find and insert`, just `insert` is OK.

Reply:
yes，i will change the code。